### PR TITLE
Update azure-site-recovery.md

### PR DESCRIPTION
**Pull request by steveswalwell**
Added a know issue of extension installation failing due to WDAC Policy being enforced

### CODE REVIEW

**Comment by Jak-MS** [comment]
@alkohli - 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/hci/manage/azure-site-recovery.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ Here's a list of known issues and the associated workarounds in this release:
...
 | 2. | Azure Site Recovery agent fails to install. No error details are seen at the cluster or server levels in the Azure Stack HCI portal. | When the Azure Site Recovery agent installation fails, it is because of the one of the following reasons:  <br><br> - Installation fails as Hyper-V isn't set up on the cluster. </br><br> - The Hyper-V host is already associated to a Hyper-V site and you're trying to install the extension with a different Hyper-V site. </br>  |
-
+| 3. | Azure Site Recovery agent fails to install. Error message of "Microsoft Azure Site Recovery Provider installation has failed with exit code - 1." appears in the portal with the failed installation. | The installation fails when WDAC is enforced. <br><br> - Setting WDAC to "Audit" mode will allow the installation to complete.  To set the WDAC mode to be Audit, you can follow the instructions in [Manage WDAC settings with PowerShell](https://learn.microsoft.com/en-us/azure-stack/hci/manage/manage-wdac#switch-wdac-policy-modes)
```
Comment (alkohli) [inline azure-stack/hci/manage/azure-site-recovery.md:245]: Can you please change the link to this relative link: [Manage WDAC settings with PowerShell](./manage/manage-wdac#switch-wdac-policy-modes).

You should have a clean build then. Thanks!

**Review by alkohli — COMMENTED** [review]
Please make the suggested change and then I will sign off. Let me know if you have any questions around this.

**Comment by steveswalwell** [comment]
@alkohli i have updated the link and the checks have now passed

**Comment by alkohli** [comment]
#sign-off

**Pull request merged**